internal/util/fetch/fetch_test.go

```diff
@@ -433,6 +427,54 @@ func TestCommand_Run_tag(t *testing.T) {
 	})
 }
 
+func TestCommand_Run_subdir_at_tag(t *testing.T) {
+	g := &testutil.TestSetupManager{
```

Comment:
We shouldn't use the `TestSetupManager` here. It creates both an upstream repo and then fetches it into the local workspace. This is useful for testing update scenarios.

For testing the fetch functionality, we should just set up an upstream repo and an empty workspace, and then explicitly use the fetch command to download the package into the workspace. This means we have to write the Kptfile into the workspace before using the fetch command. There are some other tests here that uses this pattern.

Reply:
Set up repos with `SetupReposAndWorkspace` instead.